beta-prompts/beta_prompts/improvement_engine.py

```
@@ -0,0 +1,610 @@
+            clarity_score=clarity_score,
+            specificity_score=specificity_score,
+            optimization_potential=optimization_potential,
+            recommendations=recommendations,
+        )
+
+    def optimize_prompt(
+        self,
+        prompt: str,
+        task_type: str = "general",
+        domain: str = "general",
+        enhance_reasoning: bool = True,
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)